webapp/channels/src/components/app.tsx

```diff
@@ -16,20 +16,18 @@ const LazyRoot = React.lazy(() => import('components/root'));
 
 const Root = makeAsyncComponent('Root', LazyRoot);
 
-class App extends React.PureComponent {
-    render() {
-        return (
-            <Provider store={store}>
-                <CRTPostsChannelResetWatcher/>
-                <Router history={getHistory()}>
-                    <Route
-                        path='/'
-                        component={Root}
-                    />
-                </Router>
-            </Provider>
-        );
-    }
+function App() {
+    return (
+        <Provider store={store}>
+            <CRTPostsChannelResetWatcher/>
+            <Router history={getHistory()}>
+                <Route 
+                    path="/" 
+                    component={Root} 
+                />
+            </Router>
+        </Provider>
+    );
 }
 
 export default hot(App);
```

Comment:
This was a PureComponent, so in theory it should have a `React.memo` here.
```suggestion
export default hot(React.memo(App));
```